commit c2b3384ab980f79434b1d242b258a21f91416126 (refs/changes/92/3292/5)
Author: Rob Johnston <rob.johnston@joyent.com>
Date:   2018-02-02T20:40:45+00:00 (1 year, 8 months ago)
    
    OS-6583 add missing DTDs to manlint exception list
    Reviewed by: Joshua M. Clulow <jmc@joyent.com>
    Approved by: Tim Kordas <tim.kordas@joyent.com>

diff --git a/exception_lists/manlint b/exception_lists/manlint
index 5982ac5cec..0d4ea0f464 100644
--- a/exception_lists/manlint
+++ b/exception_lists/manlint
@@ -10,10 +10,17 @@
 #
 #
 # Copyright 2016 Toomas Soome <tsoome@me.com>
+# Copyright (c) 2018, Joyent, Inc.
 #
 usr/src/boot/*
 # Not actually a manual page
+usr/src/cmd/hal/fdi/fdi.dtd.1
+usr/src/cmd/isns/isnsd/xml_def/isnsdata.dtd.1
 usr/src/cmd/svc/dtd/service_bundle.dtd.1
+usr/src/lib/fm/topo/maps/common/topology.dtd.1
+usr/src/lib/libbrand/dtd/brand.dtd.1
+usr/src/lib/libbrand/dtd/zone_platform.dtd.1
 usr/src/lib/libbsm/adt_record.dtd.1
 usr/src/lib/libbsm/adt_record.xsl.1
+usr/src/lib/libpool/dtd/rm_pool.dtd.1
 usr/src/lib/libzonecfg/dtd/zonecfg.dtd.1
